src/components/molecules/OcContextualHelper/OcContextualHelper.vue

```diff
@@ -0,0 +1,148 @@
+<template>
+  <div class="oc-contextual-helper">
+    <oc-button :id="buttonId" appearance="raw">
+      <oc-icon name="question" fill-type="line" size="small" />
+    </oc-button>
+    <oc-info-drop :drop-id="dropId" :toggle="toggleId"/>
```

Comment:
add a `v-bind="$attrs"`

Reply:
that will bind all props that you set externally on the `oc-contextual-helper` tag to the `oc-info-drop` as well.

Reply:
turns out that it needs to be `v-bind="$props"` to pass the component props down into the info drop component.